deploy/examples/crds.yaml

```diff
@@ -1296,6 +1296,9 @@ spec:
                     enabled:
                       description: Enabled represents whether the log collector is enabled
                       type: boolean
+                    maxLogSize:
+                      description: 'MaxLogSize is the maximum size of the log per ceph daemons. For example: 500M, 1G'
+                      type: string
```

Comment:
I would expect this to generate with a `pattern` field like here: https://github.com/rook/rook/blob/b0db819137799bf24d8d62126116d6c7b984fb7f/deploy/charts/rook-ceph/templates/resources.yaml#L210-L213

Maybe we need to explicitly add the regex

Reply:
Once the type is updated with `resources.Quantity`, I believe we get that regex generated automatically. @subhamkrai Have you run `make crds` since this change?